string.c

```diff
@@ -7019,12 +7019,13 @@ rb_str_include(VALUE str, VALUE arg)
  *    to_i(base = 10) -> integer
  *
  *  Returns the result of interpreting leading characters in +self+
- *  as an integer in the given +base+ (which must be in (0, 2..36)):
+ *  as an integer in the given +base+,
+ *  which must be either +0+ or in range <tt>(2..36)</tt>:
```

Comment:
Nit: Could we split this into two sentences? I think it's ambiguous whether the leading characters or `base` "must be either 0 or in range (2..36)". I actually read it as the former the first few times I read this.

Reply:
Done.